gradle-sls-packaging/build.gradle

```diff
@@ -8,8 +8,8 @@ dependencies {
 
     compile project(':sls-versions')
     compile gradleApi()
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.3.2'
-    compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.3.2' // match Gradle's version
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.11'
+    compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.11' // match Gradle's version
```

Comment:
probably delete the "match Gradle's version" comment now?

Reply:
:shrug:, I thought it was meant for people to keep in mind in the future when bumping Gradle. Can delete if you prefer.